### ISSUE TITLE
setlinewidth not working in GKSTerm

### ISSUE THREAD

**Issue by pazner** [jheinen/GR.jl#375]
`GR.setlinewidth` does not appear to have the desired effect when drawing in the GKSTerm. When writing to a file, it looks like it works properly.

For example:
```julia
GR.beginprint("out.png")
for width=1:2:7
    GR.setlinewidth(width)
    GR.polyline([0,1],[width/10,width/10])
end
GR.endprint()
```
generates the output:
![out](https://user-images.githubusercontent.com/11493037/108468946-508db300-723c-11eb-8c96-467871dee74b.png)
But in the GKSTerm window it looks like this:
<img width="612" alt="Screen Shot 2021-02-18 at 10 56 29 PM" src="https://user-images.githubusercontent.com/11493037/108469111-8f236d80-723c-11eb-9dd1-b1e42312da69.png">

GR.jl v0.54.0, Julia version info:
```
Julia Version 1.5.3
Commit 788b2c77c1 (2020-11-09 13:37 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin18.7.0)
  CPU: Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-9.0.1 (ORCJIT, skylake)
```

**Comment by jheinen** [comment]
I can't reproduce that. Which GR run-time version are you using? (`using GR; version()`)

<img width="612" alt="Screenshot 2021-02-19 at 08 28 41" src="https://user-images.githubusercontent.com/5718244/108472069-8ce8ff80-728c-11eb-819c-8b5f1eb976c3.png">

**Comment by pazner** [comment]
> I can't reproduce that. Which GR run-time version are you using? (`using GR; version()`)

0.54.0

**Comment by pazner** [comment]
Downgrading to 0.53.0 appears to work for me if I am sure to quit the GKSTerm application when changing versions.

**Comment by jheinen** [comment]
Strange. I'm on 0.54.0.

**Comment by pazner** [comment]
I checked on another Mac machine and can reproduce. It works properly on my Linux box, though.

If I change the environment variable `GKS_WSTYPE=411` to use the gksqt, it works on the Mac too, so I guess that means it's something with the Quartz GKSTerm?

**Comment by jheinen** [comment]
Could you please try the following steps:
- ensure, that the GKSTerm.app is really closed
- update GR.jl (`] up GR`)
- load the latest run-time (`ENV["GRDIR"] = ""; ] build GR`)
- check the GKSTerm checksum:
```
using GR
version()
grdir = ENV["GRDIR"]
run(`sum $grdir/Applications/GKSTerm.app/Contents/MacOS/GKSTerm`)
```
The result should be: `"0.54.0.post29"` for the version and `9583 725` for the checksum.

**Comment by pazner** [comment]
That works for me — I get the correct version and checksum, and the output in the GKSTerm is fixed.

Before, when I built GR, I did not have any `GRDIR` environment variable set, and so it downloaded "pre-compiled GR 0.54.0 Darwin binary" (version `"0.54.0"`). With `ENV["GRDIR"] = ""` it downloaded the latest binary (version `"0.54.0.post29"`), which seems to have fixed the issue.

**Comment by jheinen** [comment]
That's good news. Sorry, I just realized that the bug was fixed **after** the 0.54.0 release. We will tag a new version next week.

**Comment by pazner** [comment]
Works now with v0.55.0. Thanks!
